yarn.lock

```diff
@@ -962,13 +962,16 @@
   version "2.16.2"
   resolved "https://registry.yarnpkg.com/@octokit/types/-/types-2.16.2.tgz#4c5f8da3c6fecf3da1811aef678fda03edac35d2"
   integrity sha512-O75k56TYvJ8WpAakWwYRN8Bgu60KrmX0z1KqFp1kNiFNkgW+JW+9EBKZ+S33PU6SLvbihqd+3drvPxKK68Ee8Q==
+<<<<<<< HEAD
```

Comment:
This is clearly a merging error. This needs to be corrected. Regenerating the whole file is probably preferable.

Reply:
This comment applies for other occurrences in this file.